packages/terra-image/src/Image.jsx

```diff
@@ -59,12 +69,13 @@ const Image = ({
       alt={alt}
       height={height}
       width={width}
+      onLoad={onLoad}
+      onError={onError}
```

Comment:
I didn't know about these image events so I was posting this so other people know they exist. I think what you have is fine.